cmd/os-readdir_other.go

```diff
@@ -49,11 +50,11 @@ func readDirFn(dirPath string, filter func(name string, typ os.FileMode) error)
 		// Read up to max number of entries.
 		fis, err := d.Readdir(maxEntries)
 		if err != nil {
-			if err == io.EOF {
+			if errors.Is(err, io.EOF) {
 				break
```

Comment:
Let's avoid orthogonal changes not pertaining to the title of the PR, if not mention why are you doing these changes

Reply:
Ok, it's my carelessness, sorry for that. I already fixed it.

Also, a question is what are `orthogonal changes`? I googled it but found no relevant topic about it.

Reply:
I update the PR title to make the PR title contains my changes here.

Reply:
can you explain what error are you trying to fix here?

Reply:
> can you explain what error are you trying to fix here?

I think it's better to use `errors.Is` to judge error. So I change the code from `err == io.EOF` to `errors.Is(err, io.EOF)`